wct.config.json

```diff
@@ -1,7 +1,9 @@
 {
   "plugins":{
     "local":{
-      "browsers":["firefox"]
+      "browsers":[{
```

Comment:
This is an example of our standard wct config:

https://github.com/BrightspaceUI/status-indicator/blob/master/wct.conf.json

@svanherk : I think you ran into this sort of config with another repo.  Do you recall whether there was a reason for limiting to Firefox?